pkg/converter/annotation/annotation.go

```diff
@@ -0,0 +1,137 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package annotation
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/containerd/containerd/content"
+	"github.com/containerd/containerd/images"
+	"github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+
+	providerContent "github.com/goharbor/acceleration-service/pkg/content"
+)
+
+type Appended struct {
+	DriverType   string
+	SourceDigest string
+}
+
+const (
+	// The type annotation is used to identify different accelerated image formats in harbor.
+	// Example value: `nydus`, `estargz`.
+	AnnotationAccelerationType = "io.goharbor.artifact.v1alpha1.acceleration.type"
```

Comment:
io.goharbor.artifact.v1alpha1.acceleration.driver is better?

Reply:
should be ok.

Reply:
By the offline discussion with @wy65701436, these annotations are enough to track accelerated images in the harbor portal:

```
io.goharbor.artifact.v1alpha1.acceleration.driver.name
io.goharbor.artifact.v1alpha1.acceleration.driver.version
io.goharbor.artifact.v1alpha1.acceleration.source.digest
```

Reply:
@ktock Does estargz have any different format version?

For example, nydus have following possible values:

```
"io.goharbor.artifact.v1alpha1.acceleration.driver.version": "5"
```

```
"io.goharbor.artifact.v1alpha1.acceleration.driver.version": "6"
```

Reply:
> Does estargz have any different format version?

No, we don't as of now. :smile: